.travis.yml

```diff
@@ -20,5 +20,6 @@ before_install:
   - travis_retry composer self-update
 
 install:
+  - if [[ "$TRAVIS_PHP_VERSION" != "5.3" ]] && [[ "$TRAVIS_PHP_VERSION" != "5.4" ]]; then composer require symfony/psr-http-message-bridge:dev-wip zendframework/zend-diactoros; fi;
```

Comment:
But if we put it in `composer.json`, we would have used `require-dev` as it is no mandatory dependency.